share/mk/bsd.compiler.mk

```diff
@@ -270,19 +270,6 @@ ${X_}COMPILER_FEATURES+=	retpoline init-all
     ${TARGET_ENDIANNESS} == "1234"
 ${X_}COMPILER_FEATURES+=	compressed-debug
 .endif
-# Detect certain supported warning flags to allow building with clang versions
-# built from git between releases. For example this affects all CHERI LLVM
-# versions that include the April 2021 upstream merge but not the September one.
-# Without this check we get the following build failure:
-# error: unknown warning option '-Werror=unused-but-set-variable'
-_check_flag=${${cc}:N${CCACHE_BIN}} -Werror=unknown-warning-option -fsyntax-only -xc /dev/null
-_warning_flags_to_check=unused-but-set-variable
```

Comment:
I wonder if we should keep this just with an empty list? I feel like we might end up in a similar situation after the next merge. But I guess we can always bring it back from the git history.